server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -528,4 +559,38 @@ public CompletionStage<Void> onConfigurationEvent(ConfigurationChangedEvent even
          return eventStream.sendEvent(sse);
       }
    }
+
+   @Listener
+   public class EventLogListener {
```

Comment:
We can make it so that the `ConfigurationListener` extends this class in  order to reduce some of the repetition.